packages/react-mutation-mapper/src/component/track/OncoKbTrack.tsx

```diff
@@ -38,8 +36,9 @@ export default class OncoKbTrack extends React.Component<OncoKbTrackProps, {}> {
             return _.keys(filteredOncoKbDataByProteinPosStart)
                 .filter(position => Number(position) >= 0)
                 .map(position => ({
-                    codon: Number(position),
+                    startCodon: Number(position),
                     color: '#007FFF',
+                    itemType: TrackItemType.CIRCLE,
```

Comment:
Actually we may have multi-residue OncoKB annotations, so it might be better to decide circle vs rectangle somewhere else.